src/contributors/stevenanthonyrevo.json

```
@@ -0,0 +1,7 @@
+{
+    "name": "Steven Anthony", 
+    "desc": "A Designer and Developer with a focus on GoLang, Swift and Node",
+    "github": "https://github.com/stevenanthonyrevo",
+    "twitter": "", 
+    "linkedin": ""
```

```suggestion
    "github": "https://github.com/stevenanthonyrevo"
```
Please don't add an empty string as a value for the param. Remove it, if you don't have the link.

---

Thanks, I updated it with links!